pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -226,6 +224,12 @@ def update_train_epoch_metrics(self) -> None:
         # reset result collection for next epoch
         self.trainer._results.reset(metrics=True)
 
+    def _log_gpus_metrics(self):
+        for key, mem in self.gpus_metrics.items():
+            gpu_id = int(key.split('/')[0].split(':')[1])
+            if gpu_id in self.trainer.accelerator_connector.parallel_device_ids:
```

Comment:
Is this also set for just a single gpu?

Reply:
<img width="811" alt="Screenshot 2021-06-28 at 14 33 32" src="https://user-images.githubusercontent.com/12861981/123644984-d12d7300-d81d-11eb-91c7-7861bb1e0cf9.png">

Yes, it works.